src/routes.php

```diff
@@ -14,7 +14,7 @@
 		return view('voyager::profile');
 	});
 
-	if(Schema::hasTable('data_types')):
+	if(env('DB_CONNECTION') !== null && Schema::hasTable('data_types')):
```

Comment:
Yeah, this is a bit of a choice. It defaults to `'default' => env('DB_CONNECTION', 'mysql'),` in the default Laravel database.php config. So it might confuse people when it will still error out when you've got no .env file.